ProjectSettings/EditorBuildSettings.asset

```diff
@@ -4,112 +4,5 @@
 EditorBuildSettings:
   m_ObjectHideFlags: 0
   serializedVersion: 2
-  m_Scenes:
```

Comment:
Looks like an unintentional deletion of all scenes, Ill adjust and merge. make further changes from a branch on this one